src/Components/Message.tsx

```diff
@@ -0,0 +1,74 @@
+import { Time } from './Time';
+import React from 'react';
+import { useAppSelector } from '../redux/hooks';
+import { IConversations } from '../Interfaces/IConversations';
+import { NewMessage } from './newMessage';
+import { secondsToDate, secondsToTime } from '../utils/dateUtils';
+
+export const Message = () => {
+  const conversations = useAppSelector(
+    (state) => state.messenger.conversations
+  );
+
+  const lastNewMessage = conversations.find((conv: IConversations) => {
+    return conv.is_new ? conv.id : null;
```

Comment:
Javascript returns the first matching element from the array when you return a truthy value from its callback. So returning ID doesn't make sense.

```suggestion
    return conv.is_new;
```

Reply:
done